pages/en/learn/modules/publishing-node-api-modules.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: How to publish N-API package
-layout: docs.hbs
+title: How to publish Node-API package
+layout: learn.hbs
```

Comment:
```suggestion
layout: learn.hbs
authors: mwelwankuta
```

you can put yourself in the authors if you like

Reply:
I don't think they should add themselves to authors. They just ported the content, they didn't write the content. I would agree if they actually contributed with new text.

Reply:
I feel a link to the API (https://nodejs.org/api/n-api.html#node-api), as well as some extra notes briefly describing to the reader what a Node-API is should be included in this learn section. if i made such a change, do i qualitfy to the considered an author ? 

Will i need a saparate PR to make this change, or will this current one suffice ?

Reply:
If you're adding content, sure. But simple replacements aren't really considered content change. 

I also don't know who the original authors of this content are, and there's no authors field at the moment for these learn pages, hence I'm not sure if it does make sense adding authors at this point. I'd rather that we don't add a new authors field for now. Sorry.

Reply:
Keep in mind that the Website team is responsible by the structure of the Website, not the content. Hence why I tagged the actual team that owns this content.